File: soft2/lec01/test0.pl
sub test {
    ($i, $j) = @_;
    return ($i * $j);
}

sub main {
    $i = 3;
    $j = 2;
    $k = &test($i,$j);
    if ($k > 5) {
	print ">5\n";
    } else {
	print "<=5\n";
    }
}

& main();

File: soft2/lec01/test0.go
package main

import "fmt"

func test(i int, j int) int {
     return (i * j)
}

func main() {
     i, j := 3, 2
     var k int = 0;
     k = test(i,j)
     if k > 5 {
     	fmt.Println(">5")
     } else {
        fmt.Println("<=5")
     }
}